lib/transaction/specialtransactions/registeredtypes.js

```diff
@@ -0,0 +1,15 @@
+var registeredTransactionTypes = {
+  TRANSACTION_NORMAL : 0,
+  TRANSACTION_PROVIDER_REGISTER : 1,
+  TRANSACTION_PROVIDER_UPDATE_SERVICE : 2,
+  TRANSACTION_PROVIDER_UPDATE_REGISTRAR : 3,
+  TRANSACTION_PROVIDER_UPDATE_REVOKE : 4,
+  TRANSACTION_COINBASE : 5,
+  TRANSACTION_SUBTX_REGISTER : 8,
```

Comment:
why ints 6 & 7 not used?

Reply:
They're used in my LLMQ branch and when I'm done this will also need to be updated.